erpnext_indonesia_localization/utils/install.py

```diff
@@ -0,0 +1,104 @@
+import frappe
+from frappe.utils import now
+
+def init_setup_eil():
+	create_sales_taxes_and_charges_templates()
+
+def create_sales_taxes_and_charges_templates():
+	"""
+	Create Sales Taxes and Charges Templates with default company from global settings.
+	:return:
+	"""
+
+	global_defaults = frappe.get_single("Global Defaults")
+	default_company = global_defaults.default_company
+	company_doc = frappe.get_doc("Company", default_company)
```

Comment:
Use `frappe.db.get_value` instead of `frappe.get_doc` since this object is only called for the company abbr